File: PASCAL/Cap04/c04ex05.pas
program c04ex05;

var
  Mes: Integer;

begin
  Write('Entre com um numero equivalente a um mes: ');
  ReadLn(Mes);

  case (Mes) of
     1: WriteLn('Janeiro');
     2: WriteLn('Fevereiro');
     3: WriteLn('Marco');
     4: WriteLn('Abril');
     5: WriteLn('Maio');
     6: WriteLn('Junho');
     7: WriteLn('Julho');
     8: WriteLn('Agosto');
     9: WriteLn('Setembro');
    10: WriteLn('Outubro');
    11: WriteLn('Novembro');
    12: WriteLn('Dezembro');
  else 
    WriteLn('Mes invalido');
  end;

  WriteLn;
  Write('Tecle <Enter> para encerrar... ');
  ReadLn;
end.

File: GO/Cap04/c04ex05.go
// c04ex05.go

package main
import "fmt"

func main() {
  var Mes int

  fmt.Print("Entre com um numero equivalente a um mes: ")
  fmt.Scan(&Mes)

  switch (Mes) {
    case 1:
      fmt.Println("Janeiro")
    case 2:
      fmt.Println("Fevereiro")
    case 3:
      fmt.Println("Marco")
    case 4:
      fmt.Println("Abril")
    case 5:
      fmt.Println("Maio")
    case 6:
      fmt.Println("Junho")
    case 7:
      fmt.Println("Julho")
    case 8:
      fmt.Println("Agosto")
    case 9:
      fmt.Println("Setembro")
    case 10:
      fmt.Println("Outubro")
    case 11:
      fmt.Println("Novembro")
    case 12:
      fmt.Println("Dezembro")
    default:
      fmt.Println("Numero invalido")
  }
    
  fmt.Println()
  fmt.Print("Tecle <Enter> para encerrar... ")
  fmt.Scanln()
}
